tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -52,3 +52,21 @@ def test_small_icon_doesnt_crash(self, widget, qtbot, fake_web_tab):
 
         with qtbot.waitExposed(widget):
             widget.show()
+
+    def test_update_tab_titles_benchmark(self, benchmark, widget,
+                                         qtbot, fake_web_tab):
+        """Benchmark for update_tab_titles."""
+        widget.addTab(fake_web_tab(), 'foobar')
+        widget.addTab(fake_web_tab(), 'foobar2')
+        widget.addTab(fake_web_tab(), 'foobar3')
+        widget.addTab(fake_web_tab(), 'foobar4')
+
+        with qtbot.waitExposed(widget):
+            widget.show()
+
+        def bench():
+            for _a in range(1000):
```

Comment:
No need to do this by yourself - pytest-benchmark will [automatically adjust](http://pytest-benchmark.readthedocs.io/en/stable/calibration.html) the number of times the tests run.

Reply:
Oooh, that's pretty neat, I didn't know that! :)